src/test/java/com/redhat/cloud/common/clowder/configsource/ConfigSourceTest2.java

```diff
@@ -0,0 +1,57 @@
+package com.redhat.cloud.common.clowder.configsource;
+
+import io.smallrye.config.ConfigValue;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ *
+ */
+public class ConfigSourceTest2 {
```

Comment:
I would like to rename it to a more (domain) specific name or even put it as @Nested class in the same class as ConfigSourceTest. This test class looks a bit copy-pasted ;-)

Reply:
.